main/FatesInterfaceMod.F90

```diff
@@ -795,7 +799,16 @@ subroutine SetFatesGlobalElements(use_fates)
             max_comp_per_site = 1
          end if
             
+         ! calculate the bin edges for radiative transfer calculations
```

Comment:
@ckoven , while I feel this is as good a place as any, why did you decide to fill these arrays here? (maybe the answer is, "its as good a place as any...")

Reply:
Because I couldn't think of  a better place to do it?